### ISSUE TITLE
No force_yes parameter to pkg.upgrade 

### ISSUE THREAD

**Issue by roflmao** [saltstack/salt#21248]
Doesnt pkg.upgrade have a force_yes as pkg.install has?

```
dev-host:
    ----------
    changes:
        ----------
    comment:
        E: There are problems and -y was used without --force-yesReading package lists...
        Building dependency tree...
        Reading state information...
<snip>
       WARNING: The following packages cannot be authenticated!
          libgpg-error0
    result:
        False
```

From the docs:
```salt.modules.aptpkg.upgrade(refresh=True, dist_upgrade=True)```

**Comment by rallytime** [comment]
@roflmao Looks like it does not. If you're up to it, we would be happy to accept/review a pull request addressing this issue.

**Comment by puneetk** [comment]
@rallytime can you take a look at the associated PR. I am unsure of how to fix the https://jenkins.saltstack.com/job/PR/job/salt-pr-docs-n/797/console error, it seems unrelated to my pull

**Comment by rallytime** [comment]
Fixed via #36315
